bin/create-files.js

```diff
@@ -33,12 +35,15 @@ const readAndWriteFiles = (projectDir) => {
         console.log("Created style.css")
     })
 }
-
+// Store the path name of the executable process
 const dirToCreate = process.argv[2]
+// Current working directory
 let projectDir = process.cwd()
 if (!dirToCreate) {
+    // Execute the read-write operations in the current working directory
     readAndWriteFiles(projectDir)
 } else {
+    // Create a new project directory and execute the read-write operations in it

```

Comment:
This line is to get directory name from command line arguments,please change it accordingly